NEMO/views/users.py

```diff
@@ -481,6 +481,26 @@ def user_preferences(request):
     return render(request, "users/preferences.html", dictionary)
 
 
+@login_required
+@require_GET
+def view_user(request, user_id):
```

Comment:
you need to add a check to make sure `UserCustomization.get_bool("user_allow_profile_view")` is True, otherwise throw an error.

without it users would be able to view their profile with the URL even if the setting is not on

Reply:
added the check

Reply:
thanks!